src/app/components/OrderInput.tsx

```diff
@@ -451,13 +454,64 @@ function SubmitButton() {
 }
 
 function UserInputContainer() {
-  const { side, type } = useAppSelector((state) => state.orderInput);
+  const dispatch = useAppDispatch();
+  const { side, type, token1, token2 } = useAppSelector(
+    (state) => state.orderInput
+  );
+  const balanceToken1 =
+    useAppSelector((state) => selectBalanceByAddress(state, token1.address)) ||
+    0;
+  const balanceToken2 =
+    useAppSelector((state) => selectBalanceByAddress(state, token2.address)) ||
+    0;
+  const bestBuy = useAppSelector((state) => state.orderBook.bestBuy) || 0;
+  const bestSell = useAppSelector((state) => state.orderBook.bestSell) || 0;
 
   const isMarketOrder = type === "MARKET";
   const isLimitOrder = type === "LIMIT";
   const isBuyOrder = side === "BUY";
   const isSellOrder = side === "SELL";
 
+  useEffect(() => {
+    if (isMarketOrder) {
+      sliderCallback(0, "MARKET");
+    } else if (isLimitOrder) {
+      sliderCallback(0, "LIMIT");
+    }
+  }, [isBuyOrder, isSellOrder, isMarketOrder, isLimitOrder]);
```

Comment:
React Hook useEffect has a missing dependency: 'sliderCallback'. Either include it or remove the dependency array